scripts/ci-e2e.sh

```diff
@@ -39,7 +39,8 @@ mkdir -p "${ARTIFACTS}/logs/"
 
 ARCH=$(uname -m)
 OS=$(uname -s)
-IBMCLOUD_CLI_VERSION=${IBMCLOUD_CLI_VERSION:-"2.21.0"}
+IBMCLOUD_CLI_VERSION=${IBMCLOUD_CLI_VERSION:-"2.22.1"}
+POWER_PLUGIN_VERSION=${POWER_PLUGIN_VERSION:-"1.0.0"}
```

Comment:
but we need to tag to the older version right? otherwise you might need to fix those recent changes for the powervs command as well

Reply:
Rectified it, PTAL. Missed the change while cherry-picking it. Tagged to the older version.